.github/workflows/acceptance-tests-on-emulator.yaml

```diff
@@ -18,16 +18,23 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        ruby: [2.5, 2.6, 2.7, 3.0]
+        ruby: [2.6, 2.7, 3.0]
+        ar: [6.0.4, 6.1.4]
+        # Exclude Ruby 3.0 and ActiveRecord 6.0.x as that combination is not supported.
+        exclude:
+          - ruby: 3.0
+            ar: 6.0.4
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby
     # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby
     # (see https://github.com/ruby/setup-ruby#versioning):
       uses: ruby/setup-ruby@v1
       with:
-        bundler-cache: true
+        bundler-cache: false
         ruby-version: ${{ matrix.ruby }}
+    - name: Set ActiveRecord version
+      run: sed -i "s/\"activerecord\", \"~> 6.1.4\"/\"activerecord\", \"${{ matrix.ar }}\"/" activerecord-spanner-adapter.gemspec
```

Comment:
This overwrites the version in the gemspec file (6.1.4) with the version from the matrix.